packages/generator-volto/generators/app/index.js

```diff
@@ -123,7 +121,7 @@ Run "npm install -g @plone/generator-volto" to update.`,
 
     if (!this.voltoYarnLock) {
       this.log(chalk.red("Retrieving Volto's yarn.lock"));
-      this.voltoYarnLock = await utils.getVoltoYarnLock(voltoVersion);
+      // this.voltoYarnLock = await utils.getVoltoYarnLock(voltoVersion);
```

Comment:
Does the generator still need some work? We shouldn't be logging that it is retrieving yarn.lock if that's not what it is doing.

Reply:
True, I forgot about this one. It should be removed, we don't need that anymore, although more tests to ensure that all is in place (eg. linting and prettifying) and we don't have hoisting problems should be in place.

Reply:
I agree, the important thing is that it works and that we have tests to make sure it keeps working.